GoogleForCreators/web-stories-wp
# Deprecate `isFill` (fullBleed) setting

## Summary

Removes `isFill` setting.


## Relevant Technical Choices
N/A


## To-do

- [x] Remove `isFill` from templates.



## User-facing changes

The full-bleed icon is not available anymore for media, all the previous values have been removed.


## Testing Instructions

1. Add Media (Image and/or Video)
2. Verify that it's not possible to set the media to full-bleed.



---



Fixes #1932
